alpine/base-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# create alone user/group so we aren't forced to run app as root
```

Comment:
as mentioned on the other PR, I wouldn't create another user to run the apps, i'd just use the provided `node` user.  closer to the standard approach, less boilerplate, still the same benefits you get with creating your own user as long as you do the UID juggling on the `node` user.

Reply:
How does the "UID juggling" work?